### ISSUE TITLE
Incompatible with custom_fences

### ISSUE THREAD

**Issue by KevinRoebert** [traefik/structor#26]
Hello, we would like to set a `markdown_extensions.pymdownx.superfences.custom_fences` for our MkDocs, but unfortunately, it is not applied correctly. In our example, the `mermaid` environments are not rendered correctly anymore.

The corresponding setting can be found here: https://github.com/drasyl-overlay/drasyl/blob/master/docs/mkdocs.yml#L75

You can simply reproduce this behavior with these steps:

1. `git clone https://github.com/drasyl-overlay/drasyl.git`
2. `cd drasyl/docs`
3. 
```
structor --owner="drasyl-overlay" --repo-name="drasyl" \
            --rqts-url="https://raw.githubusercontent.com/drasyl-overlay/drasyl/master/docs/requirements.txt" \
            --dockerfile-url="https://raw.githubusercontent.com/drasyl-overlay/drasyl/master/docs/docs.Dockerfile" \
            --menu.js-url="https://raw.githubusercontent.com/drasyl-overlay/drasyl/master/docs/theme/structor-menu.js.gotmpl" \
            --exp-branch=master --force-edit-url --debug
```
4. Start a server e.g. `http-server`
5. Visit page: http://127.0.0.1:8080/architecture/diagrams/
6. Diagrams not rendered

If you try the same with `mkdocs build` or `mkdocs serve` the diagrams are rendered.

**Comment by ldez** [comment]
Hello,

Have you tried to build your doc through your Docker image?

**Comment by KevinRoebert** [comment]
> Hello,
> 
> Have you tried to build your doc through your Docker image?

Sure, you could use the Makefile as a shortcut inside the docs folder. It is using the docs.Dockerfile.

If you run `make docs` or `make docs-serve` the diagrams are rendered correctly. The structure of the documentation is pretty much the same as it is in the traefik/traefik repo.

**Comment by ldez** [comment]
it's related to the `!!python` syntax, the YAML parser that we use is not able to read those elements.

Could you try something like that:
```yml
format: '!!python/name:pymdownx.superfences.fence_div_format'
```

**Comment by KevinRoebert** [comment]
> Could you try something like that:
> 
> format: '!!python/name:pymdownx.superfences.fence_div_format'

Doesn't work. The marks breaks also the normal build with mkdocs.

**Comment by ldez** [comment]
I'm not a specialist of YAML syntax but `!!` seems related to tag syntaxt, I don't know how this work.

**Comment by KevinRoebert** [comment]
Ok. I've replaced the custom_fence with div tags. As https://github.com/readthedocs/readthedocs.org/issues/6889 and https://github.com/facelessuser/pymdown-extensions/issues/892 states, there is no real workaround if the YAML parser does not support `!!python` object creation.

**Comment by ldez** [comment]
I found a workaround for the parser, I need to do some tests, nut it's 3am for me, so I will do that tomorrow.

**Comment by ldez** [comment]
I created a fix 2623be0a2fd3f8c12a4f2b2052e8ee310c722b03

**Comment by ldez** [comment]
Fixed in v1.11.1

**Comment by KevinRoebert** [comment]
It works. Thanks a lot 👍

**Comment by KevinRoebert** [comment]
@ldez You have made a small mistake with the assets for version 1.11.1. There the checksums.txt must not contain a **v** for the version, otherwise your godownloader.sh cannot download the file anymore.

```
$ curl -sfL https://raw.githubusercontent.com/containous/structor/master/godownloader.sh | sudo bash -s -- -d -b $GOPATH/bin v1.11.1
traefik/structor info checking GitHub for tag 'v1.11.1'
traefik/structor debug http_download https://github.com/traefik/structor/releases/v1.11.1
traefik/structor info found version: 1.11.1 for v1.11.1/linux/amd64
traefik/structor debug downloading files into /tmp/tmp.SssnTW5Zrq
traefik/structor debug http_download https://github.com/traefik/structor/releases/download/v1.11.1/structor_v1.11.1_linux_amd64.tar.gz
traefik/structor debug http_download https://github.com/traefik/structor/releases/download/v1.11.1/structor_1.11.1_checksums.txt
traefik/structor debug http_download_curl received HTTP status 404
```

**Comment by ldez** [comment]
oops, I will fix that

**Comment by ldez** [comment]
Fixed in [v1.11.2](https://github.com/traefik/structor/releases/tag/v1.11.2)
